File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int oldlen = nums.length;
        int returnValue = 1;
        int[] newNums = new int[oldlen];
        int counter = 1;
        newNums[0] = nums[0];

        for (int i = 1; i < oldlen; i++) {

            if (nums[i] != nums[i - 1]) {
                returnValue++;
                newNums[counter] = nums[i];
                counter++;
            } 
        }

        System.arraycopy(newNums, 0, nums, 0, returnValue);

        return returnValue;
    }
}

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        if (len(nums) <= 0):
            return 0
        
        counter = 1

        for i in range (counter, len(nums)):
            if nums[i] != nums[i-1]:
                nums[counter] = nums[i]
                counter += 1
        
        return counter